chain/epoch_manager/src/lib.rs

```diff
@@ -992,17 +994,7 @@ mod tests {
         epoch_manager
             .record_block_info(
                 &h[1],
```

Comment:
Could you check whether you have https://github.com/nearprotocol/nearcore/blob/master/rustfmt.toml in ~/.config/rustfmt/ ?

Reply:
Nope, I don't have `~/.config/rustfmt/ `. I've started to use Idea, maybe that's why..

Reply:
FYI `~/.config/rustfmt/` is independent on IDE, it is just where rustfmt looks for config.

Reply:
of course. But who knows, maybe idea does more magic. But I checked it, rust fmt really wants to format it! You can check it by running `cargo fmt check` on staging branch.